deploy/lvm-operator.yaml

```diff
@@ -313,6 +313,12 @@ spec:
               description: VolumeGroup specifies attributes of a given vg exists on
                 node.
               properties:
+                allocationPolicy:
+                  description: 'AllocationPolicy indicates the volume group allocation
```

Comment:
why do we need map the int to value. Why can't we use the string directly here?

Reply:
To be able to plot graphs in grafana dashboards using this values. Directly string values won't be of any help for grafana panels.